### PR TITLE
App closes on pressing back from bugs fragment #281

### LINKED ISSUE #281: App closes on pressing back from bugs fragment
Describe your issue here.

### Steps to reproduce
- Select `About us` from navigation drawer
- Select `Report a Bug`
- Press back

### Expected behaviour
App should go back to about us screen

### Actual behaviour
App closes

### PR DESCRIPTION
**Pull request by syedjahangirpeeran**
Fixes #281 .

### CODE REVIEW

File: Android/app/src/main/java/io/github/project_travel_mate/utilities/BugReportFragment.java [hunk 1]
```diff
@@ -170,6 +173,28 @@ private void showProgressDialog() {
                 .progress(true, 0)
                 .show();
     }
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        if(getView() == null){
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/utilities/BugReportFragment.java]: add space after `(`

Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/utilities/BugReportFragment.java]: add space after `if`

**Comment by aayushinigam** [comment]
Issue could be solved by adding just one line of code.

**Comment by Swati4star** [comment]
@syedjahangirpeeran Look into this. It seems better to add fragment to the stack.

**Pull request closed without merging**